install.sh

```diff
@@ -329,7 +329,7 @@ EOABORT
 )"
 fi
 
-if [[ "$UNAME_MACHINE" != "x86_64" ]]; then
+if [[ -n "${HOMEBREW_ON_LINUX-}" ]] && [[ "$UNAME_MACHINE" != "x86_64" ]]; then
```

Comment:
```suggestion
if [[ "$UNAME_MACHINE" != "x86_64" ]]; then
```

This should be on all platforms (in case of e.g. running on an old PPC Mac).

Reply:
I totally forgot that PPC Macs exist :) Thanks for catching the issue.